dockerfile/build_environment.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gradle:7.1.0-jdk11 AS image
+MAINTAINER Elizaveta Zhemchuzhina, Elena Sunko, Evgeniia Kirillova
+WORKDIR /home/gradle/src
+RUN git clone -b docker-branch https://github.com/lizazhemchuzhina/SE.git .
```

Comment:
Why do you clone it? Why not just to do `ADD`...

Reply:
Fixed